ish/ish.py

```diff
@@ -18,17 +18,22 @@
     u'لأ'.lower(),     # Arabic
 ]
 
+try:
+    basestring
+except NameError:
+    basestring = (str, bytes)
 
 class TrueIsh(object):
+    def _normalize_string(s):
+        if isinstance(other, bytes):
```

Comment:
Argh, you are right. I suggest to merge #4 first, rebasing this patch on top. The change here is redundant with the code there anyway.